packages/fx-core/src/plugins/resource/frontend/utils/environment-utils.ts

```diff
@@ -21,7 +22,7 @@ export class EnvironmentUtils {
 
     let envs = "";
     for (const key in newConfigs) {
-      envs += `${key}=${newConfigs[key]}\r\n`;
+      envs += `${key}=${newConfigs[key]}${os.EOL}`;
```

Comment:
It looks the similar concat logic in packages/fx-core/src/plugins/resource/frontend/env.ts?